source/draw/RasterBgr.ooc

```diff
@@ -52,7 +52,8 @@ RasterBgr: class extends RasterPacked {
 	init: func ~allocateStride (size: IntVector2D, stride: UInt) { super(size, stride) }
 	init: func ~fromByteBufferStride (buffer: ByteBuffer, size: IntVector2D, stride: UInt) { super(buffer, size, stride) }
 	init: func ~fromByteBuffer (buffer: ByteBuffer, size: IntVector2D) { this init(buffer, size, this bytesPerPixel * size x) }
-	init: func ~fromRasterImage (original: This) { super(original) }
+	init: func ~fromRasterBgr (original: This) { super(original) }
```

Comment:
`init: func ~fromRasterBgr (original: This) { super(original) } ` will call `super(RasterPacked)`
and `init: func ~fromRasterImage (original: RasterImage) { super(original) }` will call `super(RasterImage)`. It won't compile the `convertFrom` method without the `fromRasterImage` version as it uses the base RasterImage in there.